routing/localchans/manager_test.go

```diff
@@ -258,3 +334,177 @@ func TestManager(t *testing.T) {
 		})
 	}
 }
+
+// Tests creating a new edge in the manager where the local pubkey is the
+// lexicographically lesser or the two.
+func TestCreateEdgeLower(t *testing.T) {
+	sp := [33]byte{}
+	_, _ = hex.Decode(sp[:], []byte("028d7500dd4c12685d1f568b4c2b5048e85"+
+		"34b873319f3a8daa612b469132ec7f7"))
+	rp := [33]byte{}
+	_, _ = hex.Decode(rp[:], []byte("034f355bdcb7cc0af728ef3cceb9615d906"+
+		"84bb5b2ca5f859ab0f0b704075871aa"))
+	selfpub, _ := btcec.ParsePubKey(sp[:])
```

Comment:
Nit: define all these variables in a `var` block

Reply:
They all return errors, so not sure how to do that in.a nice way.